src/clarity-icons/shapes/essential-shapes.ts

```diff
@@ -512,14 +503,10 @@ export const essentialShapes: any = {
         </svg>`,
 
     /*TODO: ellispes-vertical is deprecated and will be removed in 0.9.0*/
-    get "ellipses-vertical"() {
-        return this[ "ellipsis-vertical" ];
-    },
+
 
     /*TODO: ellispes-horizontal is deprecated and will be removed in 0.9.0*/
```

Comment:
We need to change these deprecation TODOs to 0.10.0 and make sure they carry over with the new code.